detekt-rules-style/src/main/kotlin/io/gitlab/arturbosch/detekt/rules/style/TrailingWhitespace.kt

```diff
@@ -14,6 +14,12 @@ import org.jetbrains.kotlin.psi.KtFile
 
 /**
  * This rule reports lines that end with a whitespace.
+ *
+ * Note: in KDoc comments we use Markdown, so two spaces at the end of lines should be allowed.
+ * However, JetBrains haven't implemented this in their flavour of "standard" Markdown yet (
+ * [in Dokka](https://github.com/Kotlin/dokka/issues/2823),
+ * nor [in KTIJ](https://youtrack.jetbrains.com/issue/KTIJ-6702/KDoc-Dokka-allow-for-newlines-line-breaks-inside-paragraphs)
+ * ), which means Markdown line-breaks in KDoc are really only trailing whitespace for now.
```

Comment:
```suggestion
 * However, JetBrains haven't implemented this in their flavour of "standard" Markdown yet
 * ([in Dokka](https://github.com/Kotlin/dokka/issues/2823),
 * nor [in KTIJ](https://youtrack.jetbrains.com/issue/KTIJ-6702/KDoc-Dokka-allow-for-newlines-line-breaks-inside-paragraphs)),
 * which means Markdown line-breaks in KDoc are really only trailing whitespace for now.
```

Are spaces after `(` and before `)` neccessary?

Reply:
Wanted to keep the links on separate lines, but your suggestion is rendered better and doesn't complicate the code to much